contracts/types/YieldSplitter.sol

```diff
@@ -0,0 +1,167 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.10;
+
+import {IERC20} from "../interfaces/IERC20.sol";
+import {IgOHM} from "../interfaces/IgOHM.sol";
+import {SafeERC20} from "../libraries/SafeERC20.sol";
+
+interface IOHMIndexWrapper {
```

Comment:
Would add an `@dev` comment above this explaining why this is needed vs IsOHM or something. Maybe:

```
This interface is so that we can wrap cross-chain oracles to also feed an index without needing to adhere completely to IsOHM.
```

Reply:
typo here, sorry. my fault 😅

`to also feed an index` should actually be `to feed an index` (the also shouldnt be there)